.github/workflows/precommit.yml

```diff
@@ -471,8 +471,8 @@ jobs:
             .tox/.coverage
             .tox/coverage.xml
 
-  test-snowpark-multithreading-mode:
-    name: Test Snowpark Multithreading py-${{ matrix.os }}-${{ matrix.python-version }}-${{ matrix.cloud-provider }}
+  test-snowpark-disable-multithreading-mode:
```

Comment:
should we move disable mode test to daily test?

Reply:
yeah. can do